configs/rover_configs.yaml

```
@@ -32,4 +32,28 @@ core:
     # How frequently to check the connection(second)
     check_period: ${ROVER_BACKEND_CHECK_PERIOD:5}
     # The auth value when send request
-    authentication: ${ROVER_BACKEND_AUTHENTICATION:""}
\ No newline at end of file
+    authentication: ${ROVER_BACKEND_AUTHENTICATION:""}
+
+process_discovery:
+  active: ${ROVER_PROCESS_DISCOVERY_ACTIVE:true}
+  # The period of report or keep alive process(second)
+  heartbeat_period: ${ROVER_PROCESS_DISCOVERY_HEARTBEAT_PERIOD:20}
+  # Detect processes in VM mode
+  vm:
+    # Is active the VM mode to detect processes
+    active: ${ROVER_PROCESS_DISCOVERY_VM_ACTIVE:true}
+    # The period to detect the process(second)
+    period: ${ROVER_PROCESS_DISCOVERY_VM_PERIOD:15}
+    finders:
+      # Use regex string to locate the processes
+      # Duplicate entities cannot be reported. If multiple entity are generated, only one process will be report
+      # If the multiple finders could match the same one process, only the first finder could be selected and report
+      - match_cmd_regex: ${ROVER_PROCESS_DISCOVERY_VM_FINDER_MATCH_CMD_REGEX:rover}
+        # The Layer need to relate to the process entity
+        layer: ${ROVER_PROCESS_DISCOVERY_VM_FINDER_LAYER:VM}
+        # The Service Name need to relate to the process entity
+        service_name: ${ROVER_PROCESS_DISCOVERY_VM_FINDER_SERVICE_NAME:rover}
```

```suggestion
        service_name: ${ROVER_PROCESS_DISCOVERY_VM_FINDER_SERVICE_NAME:}
```

Same as https://github.com/apache/skywalking-rover/pull/5/files#r815955089 ask, we should keep this blank, and blank should be illegal to boot.

---

Missing this?

---

Updated.